timeutil/schedule.go

```diff
@@ -557,13 +691,20 @@ func parseWeekSpan(s string) (span WeekSpan, err error) {
 		parsed.End = parsed.Start
 	}
 
-	if parsed.End.Pos < parsed.Start.Pos {
-		// eg. mon4-mon1
-		return span, fmt.Errorf("cannot parse %q: unsupported schedule", s)
-	}
+	if (parsed.Start.Pos != EveryWeek) && (parsed.End.Pos != EveryWeek) {
+		// both ends have a week position set
 
-	if (parsed.Start.Pos != EveryWeek) != (parsed.End.Pos != EveryWeek) {
-		return span, fmt.Errorf("cannot parse %q: week number must be present for both weekdays or neither", s)
+		if parsed.End.Pos < parsed.Start.Pos {
+			// eg. mon4-mon1
+			return span, fmt.Errorf("cannot parse %q: unsupported schedule", s)
+		}
+
+		if !parsed.IsSingleDay() {
+			// ambiguous case that produces different schedules depending on
```

Comment:
maybe add a comment that this will be disallowed and return an error some day as well when we drop support for "migrating" these ambiguous schedules?

Reply:
Added